tests/test_payment_intent.py

```diff
@@ -118,24 +119,24 @@ def mock_charge_get(*args, **kwargs):
         if has_account and has_customer:
 
             assert (
-                f"{pi.human_readable_amount} ({PaymentIntentStatus.humanize(fake_intent_data['status'])}) "
+                f"{get_friendly_currency_amount(pi.amount/100, pi.currency)} ({PaymentIntentStatus.humanize(fake_intent_data['status'])}) "
```

Comment:
These strings in the tests… the whole point of testing is to check that the output is what you expect. If you're just copy-pasting the behaviour of the function in the test then that defeats the whole purpose of testing.

In other words, as long as the input is static, you should test against the actual output. Something like `assert str(pi) == "some string"`

Reply:
My bad. I think I have made the same mistake in a few places. I will raise a separate PR for that as well.